capsules/README.md

```diff
@@ -93,6 +93,7 @@ These provide common and better abstractions for userspace.
 - **[Button](src/button.rs)**: Detect button presses.
 - **[Buzzer](src/buzzer_driver.rs)**: Simple buzzer.
 - **[Console](src/console.rs)**: UART console support.
+- **[HD44780 LCD](src/hd44780.rs)**: HD44780 LCD screen.
```

Comment:
Can you move this to line 48? Or if you want you can add another category under "Sensor and other IC Drivers".

Reply:
I moved it. Thank you.